future.go

```diff
@@ -9,12 +9,14 @@ import (
 
 // Future is used to represent an action that may occur in the future.
 type Future interface {
-	// Error blocks until the future arrives and then
-	// returns the error status of the future.
-	// This may be called any number of times - all
-	// calls will return the same value.
-	// Note that it is not OK to call this method
-	// twice concurrently on the same Future instance.
+	// Error blocks until the future arrives and then returns the error status
+	// of the future. This may be called any number of times - all calls will
+	// return the same value.
+	// Note that it is not OK to call this method twice concurrently on the same
+	// Future instance.
+	// Note that Error() will only return generic errors related to raft, such
```

Comment:
Minor itsy bitsy spider sized nit: You can just remove the Note (or both of them) and just keep that as part of the informational comment.